tests/containers/bci_repo.pm

```diff
@@ -42,41 +42,55 @@ my $tdata = [
     }
 ];
 
+my @pkg_regex = ('kernel-default', 'yast', 'gnome-desktop', 'qemu-kvm');
+
 sub container_exec {
     my $container = shift;
     my $cmd = shift;
     assert_script_run(qq[podman exec $container /bin/sh -c '$cmd'], @_);
 }
 
+sub prepare_repo {
+    my $container = shift;
+    my $bci_repo = get_required_var('REPO_BCI');
+    container_exec($container, 'zypper ref', timeout => 180);
+    container_exec($container, 'zypper lr -d');
+    container_exec($container, "zypper -q -s 11 pa --orphaned | tee -a repo.org", timeout => 600);
+    # remove container-suseconnect to not get packages from registered SLE host
+    container_exec($container, 'zypper -n rm container-suseconnect', timeout => 180);
+    # refresh services in order to remove container-suseconnect-zypp orphaned services
+    container_exec($container, 'zypper refs');
+    # remove SLE_BCI repo pointing to official update servers and add SUT repo
+    container_exec($container, 'zypper rr 1');
+    container_exec($container, "zypper ar http://openqa.suse.de/assets/repo/$bci_repo BCI_TEST");
```

Comment:
Isn't there a function to get the asset URL?

Reply:
Not sure, but this variable comes from OBS sync var.

Reply:
Apparently `data_url("REPO_BCI")` works

Reply:
Actually, only almost. It only matches `REPO_\d`...